docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-inline-video/index.scss

```diff
@@ -0,0 +1,80 @@
+@use 'uswds-core' as *;
+
+.ncids-inline-video {
```

Comment:
The style naming here is all over the place. The block element should match the component name, so `.cgdp-inline-video`. 

In thinking about this, the NCIDS inline video paragraph on the home and landing, is the specific implementation here. It will have analytics directly related to home and landing pages. So there is probably another `cgdp-video` component that needs to be added to components for the general styling of an embedded video. That can have variants of 100%, 40%, etc when the time comes. This cgdp-inline-video should really only have the styling for the home/landing container.